resources/lang/en/events/relationship_events/breakup_mates.json

```
@@ -0,0 +1,38 @@
+{
+	"had_fight":[
+		"m_c and r_c had a huge fight and broke up.",
+		"At the same spot in camp they announced their commitment to each other as mates, m_c and r_c announce their break up, looking coldly at each other with anger and hurt in their eyes.",
+		"m_c and r_c have been arguing with each other a lot as of late, making everyone uneasy around them. c_n is secretly relieved when they announce that they're no longer mates.",
+		"m_c and r_c had an excellent relationship and were always at each other's side. But to the Clan's surprise, m_c and r_c had a huge fight in the middle of the clearing, exchanging hurtful words before calling off their relationship for good.",
```

```suggestion
		"m_c and r_c had a huge fight in the middle of the clearing, exchanging hurtful words before calling off their relationship for good.",
```
We can't really guarantee that the words I removed are true. If this pair actually had a rocky relationship, this event might feel odd to the player.